### PR TITLE
[COPS-7154] 2.1.1 Air-Gapped Bundles v2

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7154

## Description of changes being made

Repurposed AWS Govcloud sections and added new material to it. Here's try #2!

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4220.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — CHANGES_REQUESTED** [review]

**Review by cmastr — APPROVED** [review]

**Comment by dkoshkin** [comment]
@mwrightd2 please take a look at the rendered pages and  http://docs-d2iq-com-pr-4220.s3-website-us-west-2.amazonaws.com/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/  (some already suggested by @cneth )

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+## Configure the bootstrap cluster
+
+1. Set the cluster name using environment variables.
+
+    ```bash
+  # Clustername
+  CLUSTER_SUFFIX=ABC
+  export CLUSTER_NAME=cluster-${CLUSTER_SUFFIX,,}
+    ```
```
[the remark this answers is not in the document]
Comment (mwrightd2) [reply pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md]: @cneth so if leaving this in here at all, just move to end of doc like a "hey if you want to do this, you can"

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+menuWeight: 10
+excerpt: Configure, set up the control plane for, and create your bootstrap cluster.
+beta: false
+enterprise: false
+---
+This section covers the configuration and creation of a bootstrap cluster and it's control plane.
+
+## Configure the bootstrap cluster
+
+1.   To make the following examples easier to follow, set an environment variable with the cluster name.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:14]:
```suggestion
1.  Set an environment variable with the cluster name. to make the following examples easier to follow.
```

Comment (Colin1841) [reply pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:14]: This had 3 spaces between 1.   and set - is that correct, or is single space, like the following steps below, correct?

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+## Create the bootstrap cluster
+
+1. Create the bootstrap cluster.
+
+```bash
+dkp create bootstrap
+```
+
+1. Once the cluster has been created, Generate the cluster yaml.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:61]:
```suggestion
1. Once the cluster is created, Generate the cluster yaml.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+dkp get kubeconfig -c cluster-abc > cluster-abc.conf
+```
+
+1.  Verify the nodes are ‘Ready’.
+
+```bash
+kubectl --kubeconfig cluster-abc.conf get nodes
+```
+
+1.  Add the ClusterAPI controllers to the cluster
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:107]:
```suggestion
1.  Add the ClusterAPI controllers to the cluster.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+```bash
+dkp create bootstrap controllers --with-aws-bootstrap-credentials=false --kubeconfig cluster-abc.conf
+```
+
+1.  When the workload cluster is ready, move the cluster lifecycle services to the workload cluster.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:113]:
```suggestion
1.  Move the cluster lifecycle services to the workload cluster, when the workload cluster is ready, 
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+```bash
+dkp move --to-kubeconfig cluster-abc.conf
+```
+
+1.  Wait for the cluster control-plane to be ready
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md:119]:
```suggestion
1.  Wait for the cluster control-plane to be ready.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/createami/index.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      - docker.io/mesosphere/pause-alpine:3.2
+      - docker.io/mesosphere/dkp-diagnostics-node-collector:v0.3.3
+    ```
+
+    ```bash
+     konvoy-image build --ami-regions us-west-1,us-east-1 --region us-east-1 --source-ami ami-092e75227e47facfc --overrides overrides/custom.yaml --overrides overrides/images.yaml images/ami/centos-7.yaml
+
+    ```
+
+Then, create your bootstrap cluster.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/createami/index.md:92]:
```suggestion
1.  Create your bootstrap cluster.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/index.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+layout: layout.pug
+navigationTitle: Air-Gapped Bundles
+title: Air-Gapped Bundles
+menuWeight: 10
+excerpt: Air-Gapped Bundles
+beta: false
+enterprise: false
+---
+
+DKP comes with an easy way to deploy in an air-gapped environment. Rather than one large package that includes the requirements for all air-gapped environments, D2iQ now provides air-gapped bundles for individual needs. These sections cover the download, environment setup, and bootstrap cluster creation for air-gapped bundles.
```
[the remark this answers is not in the document]
Comment (Colin1841) [reply pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/index.md:11]: Leave the D2i reference if you like it in here, nothing wrong with it, just looks a little clunky to me.

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+* Load and create a local registry
+* Update Docker configuration with registry reference
+* Seed a local registry
+* Load the bootstrap image
+
+## Prerequisites
+
+* Docker CE requires a minimum version of 19.03
+
+## Setup docker
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md:22]:
```suggestion
## Set up docker
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md [hunk 10]
```diff
@@ -0,0 +1,189 @@
+---
+layout: layout.pug
+navigationTitle: Load Bootstrap Image
+title: Load Bootstrap Image
+menuWeight: 10
+excerpt: Setup docker in your environment, configure your local registry, and load the bootstrap image.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md:6]:
```suggestion
excerpt: Set up docker in your environment, configure your local registry, and load the bootstrap image.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+done < <(tar xfO "${AIRGAPPED_TAR_FILE}" "index.json" | grep -oP '(?<="io.containerd.image.name":").*?(?=",)')
+
+echo "Loaded ${AIRGAPPED_TAR_FILE}"
+EOF
+
+seed.sh ${KOMMANDER_AIRGAPPED_TAR_FILE}
+```
+## Load the bootstrap image and verify the contents
+
+1. Once the registry has been seeded, load the bootstrap image.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/loadbootstrap/index.md:85]:
```suggestion
1. Load the bootstrap image, after the registry is seeded.
```

**Review by Colin1841 — APPROVED** [review]

**Comment by mwrightd2** [comment]
Pulling down for: https://github.com/mesosphere/dcos-docs-site/pull/4270

**Pull request closed without merging**